packages/nimble-components/src/utilities/style/user-select.ts

```diff
@@ -0,0 +1,6 @@
+import { cssPartial } from '@microsoft/fast-element';
+
+export const disableSelect = cssPartial`
```

Comment:
nit: can we keep a name that aligns with the css since it's just a text replacement for that, i.e. `userSelectNone`

Reply:
That makes sense. I've renamed `disableSelect` to `userSelectNone`.